modules/components/stories/Portal.js

```diff
@@ -158,9 +158,6 @@ storiesOf('Portal', module).add('Exploration', () => (
                             .map(agg => ({
                               ...agg,
                               ...data[index].aggregations[agg.field],
-                              ...data[index].extended.find(
```

Comment:
is this on purpose?

Reply:
This is part of the normalization commit